src/firefly/java/edu/caltech/ipac/firefly/server/visualize/PushCommands.java

```diff
@@ -311,14 +312,52 @@ public String doCommand(Map<String, String[]> paramMap) throws Exception {
 
         public String doCommand(Map<String, String[]> paramMap) throws Exception {
             SrvParam sp= new SrvParam(paramMap);
-            String ip= sp.getRequired(ServerParams.IP_ADDRESS);
-            boolean active= PushJob.isBrowserClientActive(ip);
+            String channel= sp.getOptional(ServerParams.CHANNEL_ID);
+            boolean active= PushJob.getBrowserClientActiveCount(channel,0)>1;
```

Comment:
What happens if channel is null? Is eventChannel in RequestOwner is always non-null?